solutions/src/test/java/ru/sberbank/school/task08/JacksonManagerTest.java

```diff
@@ -0,0 +1,45 @@
+package ru.sberbank.school.task08;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import ru.sberbank.school.task08.state.GameObject;
+import ru.sberbank.school.task08.state.InstantiatableEntity;
+import ru.sberbank.school.task08.state.MapState;
+import ru.sberbank.school.task08.state.Savable;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class JacksonManagerTest {
+
+    private static JacksonManager manager;
+    private static MapState<GameObject> mapState;
+    private static String filePath = "C:\\test";
+    private static String fileName = "jackson";
+
+    @Before
+    public void init() {
+        manager = new JacksonManager(filePath);
+        manager.initialize();
+        List<GameObject> gameObjectList = new ArrayList<>();
+        gameObjectList.add((GameObject) manager.
```

Comment:
&#39;.&#39; should be on a new line.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a271036570>*